Sources/SwiftDocC/Infrastructure/Workspace/DocumentationBundle+Info.swift

```diff
@@ -271,6 +282,7 @@ extension BundleDiscoveryOptions {
         fallbackDefaultCodeListingLanguage: String? = nil,
         fallbackDefaultModuleKind: String? = nil,
         fallbackDefaultAvailability: DefaultAvailability? = nil,
+        fallbackFeatureFlags: DocumentationBundle.Info.BundleFeatureFlags? = nil,
```

Comment:
Why do we need to add this argument? We don't have a CLI fallback for this so the caller would always pass `nil`.

Reply:
Strictly speaking, it's not _required_, but i added it for completeness. I briefly added a CLI fallback but took it out early on in development.